Tasks/JenkinsDownloadArtifacts/tslint.json

```diff
@@ -0,0 +1,60 @@
+{
+  "rules": {
```

Comment:
What causes this tslint.json to be used?  I would prefer that we share a tslint.json in this repo with a way for a specific task to opt out (and perhaps that's the initial setting for all the other tasks).  Then each team does work to fix lint issues and remove the opt out.

What I'm after is consistent engineering in our team